pyro/infer/elbo.py

```diff
@@ -45,6 +45,8 @@ class ELBO(object):
         be ignored. Defaults to False.
     :param bool retain_graph: Whether to retain autograd graph during an SVI
         step. Defaults to None (False).
+    :param float tail_adaptive_beta: Exponent beta with -1.0 >= beta > 0.0 for
```

Comment:
`>` -> `<`?

Reply:
whoops

Reply:
Also you'll probably want to enclose these in double ticks
```
Exponent beta with ``-1 <= beta < 0`` for ...
```

Reply:
done